### PR TITLE
Added caret-up-down to select your language on main page

### LINKED ISSUE #662: Add caret up-down for select your language on main page
### Expected results
There should be caret up-down for select your language
### Actual results
No such property found.
### Implementation/Solution Ideas
Use javascript.
### Would you like to work on this
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [ ] Squashed the commits.

### Changes done in this Pull Request
link-  http://rawgit.com/harsh-jindal/labyrinth/caret-up-down/index.html
-Fixes #662
### My steps to solve it
Used JQuery to solve it.
Files Changed: index.html 
### Screenshots,
screen shot 2018-01-25 at 12 57 38 am

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -295,6 +296,15 @@ <h1 class="brand-logo center">Labyrinth</h1>
...
+  <!-- Caret-up-down -->
+  <script>
+    var $z=$('.select-your-language>a');
+    var $a=$('.languagepicker.roundborders.large')
+      $a.hover(function(){
+        $z.html('Select your language <span class="fa fa-caret-up"></span>');
+      },function(){
+        $z.html('Select your language <span class="fa fa-caret-down"></span>');
+      })
+  </script>
```
Comment (niccokunzmann) [inline index.html]: Thanks!
I think, you can do this with the :hover css property. Would you like to try it?
Also, adding Javascript at the end of the file is not the style we go for. For consistency, that would go to the js folder if it is still needed.

Comment (niccokunzmann) [reply index.html]: there is also :after to add text.

Comment (harsh-jindal) [reply index.html]: @niccokunzmann I'll add the javascript to sidebar.js 
Is that fine?

**Comment by niccokunzmann** [comment]
I do not know what is in the file but I would say, sure!
Also, please have a look at what we do with
window.addEventListener("load", function(){....})
The dom is not loaded, when the scripts are loaded so that they need to
delay access to the DOM.

File: index.html (hunk 1 above)
Comment (niccokunzmann) [reply index.html]: You can do that, sounds ok. If the DOM is not loaded when the script is executed,
you can use `document.addEventListener("load", function (){ your code goes here})`.

Comment (niccokunzmann) [reply index.html]: looks like emails can not reply to these comments.

Comment (harsh-jindal) [reply index.html]: Okay @niccokunzmann I'll make the changes

**Comment by harsh-jindal** [comment]
@niccokunzmann Made the changes. Didn't need to use window.addEventListener("load", function(){....}).

**Comment by niccokunzmann** [comment]
Okay then. travis is failing. Could you check?

**Comment by niccokunzmann** [comment]
https://travis-ci.org/fossasia/labyrinth/builds/332990257#L537

**Comment by harsh-jindal** [comment]
@niccokunzmann I have checked that and made the changes.

**Comment by niccokunzmann** [comment]
Thanks!

**Pull request merged**